packages/scripts/webapp/preset/config/webpack.config.js

```diff
@@ -108,8 +108,14 @@ module.exports = ({ getUserConfig, mode }) => {
 				{
 					test: /\.scss$/,
 					use: getSassLoaders(true, sassData, mode),
+					include: /@talend/,
 					exclude: /bootstrap-theme/,
 				},
+				{
+					test: /\.scss$/,
+					use: getSassLoaders(getUserConfig(['sass', 'modules'], true), sassData, mode),
+					exclude: /@talend/,
```

Comment:
Wondering if this should also exclude `bootstrap-theme` since it's already covered on line 106, and it's excluded when modules is set to true above?

Reply:
Yeah never mind, as you said "bootstrap-theme" is inside a "@talend" module so it seems good